api/policies/v1alpha2/clusteradmissionpolicy_types.go

```diff
@@ -77,12 +77,13 @@ type ClusterAdmissionPolicySpec struct {
 // ClusterAdmissionPolicy is the Schema for the clusteradmissionpolicies API
 // +kubebuilder:object:root=true
 // +kubebuilder:subresource:status
-// +kubebuilder:resource:scope=Cluster
+// +kubebuilder:resource:scope=Cluster,shortName=cap
```

Comment:
@kubewarden/kubewarden-developers I've needed to update v1alpha2 CRDs because controller-gen change the order of it process the files to extract the kubebuilder markers each time it runs. Therefore, each time it runs the CRDs generation is different. When the v1alpha2 CRDs types are the last one to be processed, the shortNames are not define.

Reply:
I've opened an [issue](https://github.com/kubernetes-sigs/controller-tools/issues/1068) asking if this is expected or it is a bug.

Reply:
Can't we drop these alpha resources? They are not used since a long time...

Reply:
+1 on dropping the alpha resources.

Reply:
I thoug about that as well. I've even use the `//+kubebuilder:skip` to ignore the `v1alpha2`. But than I've wondered, what happens if someone is still using the alpha version? The conversion will happen out of the box for resources already installed? Because of that, I've decided to mark the alpha version as deprecated first. But if we can drop it with no issue, I prefer removing it as well.

Reply:
Considering the previous comment. I suggest to keep the `v1alpha2` for now, mark it as deprecated and create an issue to attack the old CRD removal.